lib/hunter/node.rb

```diff
@@ -46,16 +47,17 @@ def remove_groups(to_remove)
       @groups = @groups - to_remove
     end
 
-    attr_reader :id, :ip, :payload, :groups, :hostname, :label
-    attr_accessor :label
+    attr_reader :id, :ip, :payload, :groups, :hostname, :label, :prefix
+    attr_accessor :label, :prefix
```

Comment:
You don't need to have `prefix` in both of these (`label` shouldn't really be in both, either). `attr_accessor` is shorthand for `attr_reader` and `attr_writer`. Please could you move them both to just `attr_reader`, unless you can think of a reason why we might want to mutate them at runtime.

Reply:
`label` actually is mutated at runtime, during the `parse` command, at least in the implementation used in the current `master` branch. I had planned to do something similar with `prefix` but it wasn't necessary in the end, so I'll keep `label` as an `attr_accessor` for now for the sake of consistency, although it sounds like significant changes are coming to the `parse` command so this may not be relevant soon.